### DISCUSSION TITLE
What is the ScriptLoaded component used to Wrap GoogleMap component like for GroundOverlay example ?

### DISCUSSION THREAD

**Discussion by JulienDesmettre** [JustFly1984/react-google-maps-api#2965] [Q&A]
Hello, i'm trying to set a GroundOverlay component as you do here in your doc: [https://react-google-maps-api-docs.netlify.app/#groundoverlay ](https://react-google-maps-api-docs.netlify.app/#groundoverlay) 

If i try to require `const ScriptLoaded = require('../../docs/ScriptLoaded').default;` that's throw an error when running Gatsby dev "Module not found: Can't resolve '../../docs/ScriptLoaded" ( but no error in my TS server...??? ).

As is, if i don't wrap my div in ScriptLoaded component but in LoadScript component, the GoogleMap works , but not the GroundOverlay. Does the ScriptLoaded component is compulsory to make the GroundOverlay appear ? What is this component ( i don't find anything on Google ).

Thanks in advance for your help.

**Comment by ysageev** [comment]
I have the same question.

**Reply by JulienDesmettre** [reply]
Still not sure what is the Scriptloaded component, but it works with the LoadScript or LoadScriptNext components ( to wrap the GoogleMap component ) , and these two components come with the Library , so it doesn't throw any error if you import it like GoogleMap component. So you can replace Scriptloaded by LoadScript or LoadScriptNext. Then i figured out why the GroundOverlay component wasn't visible : if you use the exact bounds and the exact center value given in the example it will work. As far as i was concerned, i tried with some other values, and it seems that it has to be very precise to work. I'm not sure the way you can do to get the exact tiles bounds you need so that works perfectly.
